File: wenger-common/src/main/java/io/github/rcarlosdasilva/wenger/common/constant/GeneralConstant.java
package io.github.rcarlosdasilva.wenger.common.constant;

import java.nio.charset.Charset;

/**
 * @author <a href="mailto:rcarlosdasilva@qq.com">Dean Zhao</a>
 */
public class GeneralConstant {

  public static final String DEFAULT_APPLICATION_NAME = "wenger";
  public static final String DEFAULT_ENCODING = "UTF-8";
  public static final Charset DEFAULT_CHARSET = Charset.forName(DEFAULT_ENCODING);

  public static final String URL_SEPARATOR = "/";

  private GeneralConstant() {
    throw new IllegalStateException();
  }

}

File: wenger-common/src/main/java/io/github/rcarlosdasilva/wenger/common/constant/GeneralConstant.kt
package io.github.rcarlosdasilva.wenger.common.constant

import java.nio.charset.Charset

/**
 * @author [Dean Zhao](mailto:rcarlosdasilva@qq.com)
 */
class GeneralConstant private constructor() {

  companion object {

    const val DEFAULT_APPLICATION_NAME = "wenger"
    const val DEFAULT_ENCODING = "UTF-8"
    val DEFAULT_CHARSET = Charset.forName(DEFAULT_ENCODING)

    const val URL_SEPARATOR = "/"
  }

}
